politico/api/v1/party/modal.py

```diff
@@ -0,0 +1,120 @@
+"""
+    contains the class for party modal
+"""
+
+class Party:
+    def __init__(self, id, name, hqAddress, logoUrl):
+        self.id = id
+        self.name = name
+        self.hqAddress = hqAddress
+        self.logoUrl = logoUrl
+
+    @property
+    def id(self):
+        return self.id
+
+    @id.setter
+    def id(self, id):
+        self.id = id
+
+    @property
+    def name(self):
+        return self.name
+
+    @name.setter
+    def name(self, name):
+        self.name = name
+
+    @property
+    def hqAddress(self):
+        return self.hqAddress
+
+    @hqAddress.setter
+    def hqAddress(self, hqAddress):
+        self.hqAddress = hqAddress
+
+    @property
+    def logoUrl(self):
+        return self.logoUrl
+
+    @logoUrl.setter
+    def logoUrl(self, logoUrl):
+        self.logoUrl = logoUrl
+
+
+    @property
+    def party_data(self):
+        party_data = {}
+        party_data['id'] = self.id
+        party_data['name'] = self.name
+        party_data['hqAddress'] = self.hqAddress
+        party_data['logoUrl'] = self.logoUrl
+        return party_data
+
+    @property
+    def party_patch(self):
+        party_patch = {}
+        party_patch['id'] = self.id
+        party_patch['name'] = self.name
+        return party_patch
+
+
+class partyTable:
+    """ acts as a table for storing parties and their related information"""
+
+    def __init__(self):
+        pass
+
+    parties = [
```

Comment:
don't hardcode data, leave this as a blank list that can be populated later on memory
```suggestion
    parties = []
```